hips/tiles/description.py

```diff
@@ -0,0 +1,99 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Classes and functions to manage HiPS tiles."""
+
+
+from collections import OrderedDict
+
+__all__ = [
+    'HipsDescription',
+]
+
+class HipsDescription:
+    """This class provides methods for parsing the HiPS properties file.
+
+    It also provides multiple getters around the commonly used HiPS properties.
+
+    Parameters
+    ----------
+    properties : str
+        A string containing HiPS tile properties
+    """
+
+    def __init__(self, properties: str) -> None:
+        super(HipsDescription, self).__init__()
```

Comment:
`HipsDescription` has no super-class (other than the default `object` in Python). No need to call `super` here, you can remove this line.